server.py

```diff
@@ -31,14 +31,29 @@ def api_book_postings(b):
     return json.dumps([])
 
 
+@app.route('/api/book/<b>/postings/new')
```

Comment:
It might be a good idea to add version numbers to each endpoint for backwards compatibility i.e. `/api/v1`